packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -677,6 +677,12 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
   SemanticsHandle ensureSemantics() {
     return binding.pipelineOwner.ensureSemantics();
   }
+
+  /// Given a widget `W` specified by [finder] and a [Scrollable] widget `S` in
+  /// its ancestry tree, this scrolls `S` so as to make `W` visible.
+  ///
+  /// Shorthand for `Scrollable.ensureVisible(tester.element(finder))`
+  Future<Null> ensureVisible(Finder finder) => Scrollable.ensureVisible(element(finder));
```

Comment:
Analysis is failing because `Future<void>` is the modern replacement for `Future<Null>`

Reply:
Thanks, I was just trying to figure it out.